featuretools_tsfresh_primitives/utils.py

```diff
@@ -1,45 +1,75 @@
-import numpy as np
-import pandas as pd
+from featuretools.primitives import AggregationPrimitive, TransformPrimitive
+from tsfresh.feature_extraction.settings import ComprehensiveFCParameters
 
 import featuretools_tsfresh_primitives
 
 
-def _pascal_case(snake_str):
-    words = snake_str.split('_')
-    titled_words = map(str.title, words)
-    return ''.join(titled_words)
+def comprehensive_fc_parameters():
+    """It is basically a dictionary, which is a mapping from string to a dictionary list of parameters.
```

Comment:
Let's update this docstring to explain that this is a wrapper around ComperehensiveFCParameters to filter out unsupported parameter settings.